packages/document/module-doc/docs/en/plugins/official-list/plugin-babel.mdx

```diff
@@ -25,6 +25,34 @@ export default defineConfig({
 });
 ```
 
+:::tip
+You can also configure the registration via hooks, for example,
+if you need to bundle two files A and B at the same time and only need to use babel when bundle A:
+
+```ts
```

Comment:
A "tip" should not contain a long code block, is it ugly...

It can just be a section rather than a tip.